libclc/CMakeLists.txt

```diff
@@ -261,10 +261,13 @@ foreach( t ${LIBCLC_TARGETS_TO_BUILD} )
 	foreach( d ${${t}_devices} )
 		# Some targets don't have a specific GPU to target
 		if( ${d} STREQUAL "none" OR ${ARCH} STREQUAL "spirv" OR ${ARCH} STREQUAL "spirv64" )
-			set( mcpu )
+			# FIXME: Ideally we would not be tied to a specific PTX ISA version
+			if( ${ARCH} STREQUAL nvptx OR ${ARCH} STREQUAL nvptx64 )
+				set( flags "SHELL:-Xclang -target-feature" "SHELL:-Xclang +ptx64")
```

Comment:
Why using `"SHELL:` and `string( REGEX REPLACE "SHELL:"` later is needed ?

Reply:
`add_target_options` only works if the `SHELL:` is there, but `add_custom_command` only works if the `SHELL:` is *not* there.

This is definitely a bit of a hack, but it seemed less error-prone than defining the same set of flags twice.  If there's a more standard way to do this, please let me know and I'll fix it.

Reply:
Makes sense. I'm no CMake expert so I'm not quite sure how to make it better.